### ISSUE TITLE
Possible for a view to show a thumbnail image of other views

### ISSUE THREAD

**Issue by echoleaf** [tokenshift/obsidian-page-gallery#65]
I love that PG is letting me ditch Pinterest. Is it possible for a view to show thumbnail images associated with other views, like how Pinterest displays boards?

**Comment by tokenshift** [comment]
Like a single view/tab whose tiles are "previews" of the other views in the gallery? I'll have to think about how to add something like that.

**Comment by echoleaf** [comment]
Yes, and I'm specifically thinking of Pinterest. I wouldn't say this is super important, as the tabs do make switching to a 'subview' very easy. My workaround for now is to have the first view show notes from all 'subviews,' except I named it 'New' and limited the number of visible notes.
